addons/l10n_cl/models/l10n_latam_document_type.py

```diff
@@ -13,7 +13,12 @@ class L10nLatamDocumentType(models.Model):
             ('invoice_in', 'Purchase Invoices'),
             ('debit_note', 'Debit Notes'),
             ('credit_note', 'Credit Notes'),
-            ('receipt_invoice', 'Receipt Invoice')])
+            ('receipt_invoice', 'Receipt Invoice'),
+            ('stock_picking', 'Stock Delivery'),
+        ],
+    )
+    l10n_cl_active = fields.Boolean(
+        'Active in localization', help='This boolean enables document to be included in journals')
```

Comment:
to be included on invoices  (might be clearer than on journals)

Reply:
ok! 7a4dec8b56fce2accdb7048da63c2ccb901897e8